mlir/include/mlir/Target/Wasm/WasmBinaryEncoding.h

```diff
@@ -0,0 +1,73 @@
+//===- WasmBinaryEncoding.h - Byte encodings for Wasm binary format ===----===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Define encodings for WebAssembly instructions, types, etc from the
```

Comment:
Minor nit: s/`etc`/`etc.`
(other: `from the WebAssembly` or `_for_ the WebAssembly`?)